packages/utils/package.json

```
@@ -43,7 +43,8 @@
     "@walletconnect/window-getters": "1.0.1",
     "@walletconnect/window-metadata": "1.0.1",
     "detect-browser": "5.3.0",
-    "uint8arrays": "5.1.0"
+    "uint8arrays": "5.1.0",
+    "elliptic": "^6.5.7"
```

```suggestion
    "elliptic": "6.5.7"
```

Fixed versioning + update package-lock.json 🙏